src/ArraySetops.chpl

```diff
@@ -267,8 +275,8 @@ module ArraySetops
             twoArrayRadixSort(tmp, new KeysRanksComparator());
 
             const writeToResultSlice = (segs[here.id]+numSendBack)..#tmp.size;
-            sortedIdx[writeToResultSlice] = [(key, val) in tmp] key;
-            permutedVals[writeToResultSlice] = [(key, val) in tmp] val;
+            sortedIdx[writeToResultSlice] = [(key, _) in tmp] key;
+            permutedVals[writeToResultSlice] = [(_, val) in tmp] val;
```

Comment:
Is there any advantage in using `_` here?

Reply:
It's just stylistic! It makes it easier to tell at a glance which looping variables are actually being used

this was one of the style suggestions I got from the `chpl_check` linter